drucker_dashboard/apis/__init__.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+
+
+import traceback
+
+from flask_restplus import Api
+from flask_restplus.utils import default_id
+from kubernetes.client.rest import ApiException
+
+try:
+    from my_dashboard_logger import logger
```

Comment:
I want to add a custom logger for users.
Do you know how to realize it?

Reply:
I just wonder if it is better to add document about how to add a custom logger because there is functionality to add the one, but there is no instruction about it.

Reply:
I add the functionality of specifying user logger.
See https://github.com/rekcurd/drucker-dashboard/pull/54/commits/a104b94e334f2afd772ea7b041b6228b48bc16d4